gateleen-routing/src/main/java/org/swisspush/gateleen/routing/CustomHttpResponseHandler.java

```diff
@@ -35,8 +34,8 @@ public boolean handle(HttpServerRequest request) {
         try {
             String code = request.uri().substring(path.length());
 
-            List<String> pathSegments = Lists.newArrayList(Splitter.on(SLASH).omitEmptyStrings().limit(2).split(code));
-            code = pathSegments.isEmpty() ? EMPTY : pathSegments.get(0);
+            final Iterator<String> pathSegments = Splitter.on(SLASH).omitEmptyStrings().limit(2).split(code).iterator();
```

Comment:
Uh - I only see now that this is Splitter from Guava - aaargh. Massive fat library for stuff which is (mostly) part of Java 8 :-/

Reply:
Already removed. Have another look